app/views/exceptions/not_found.html.erb

```diff
@@ -9,11 +9,48 @@
+<path d="M10.6672 15.165C10.8922 15.5133 10.7695 15.9899 10.4043 16.1864C10.2331 16.2875 10.0339 16.3088 9.85236 16.2621C9.64809 16.2095 9.48403 16.0947 9.3829 15.9234L7.61313 12.9267L4.59371 14.6906C4.42247 14.7918 4.22336 14.8131 4.04178 14.7664C3.83751 14.7138 3.67345 14.599 3.57232 14.4277C3.34737 14.0794 3.47006 13.6027 3.83524 13.4063L6.83781 11.6139L5.06804 8.61714C4.84308 8.26882 4.96577 7.79217 5.33095 7.59575C5.67928 7.37079 6.15593 7.49349 6.35234 7.85867L8.14481 10.8612L11.1415 9.09146C11.4899 8.86651 11.9665 8.9892 12.1629 9.35438C12.3879 9.70271 12.2652 10.1794 11.9 10.3758L8.89744 12.1682L10.6672 15.165Z" fill="#var(--tertiary-high)"/>
+<circle cx="73" cy="47" r="46" fill="var(--secondary)" stroke="var(--tertiary-high)" stroke-width="2"/>
+<rect x="101" y="85.2603" width="14" height="31" transform="rotate(-47.1283 101 85.2603)" fill="var(--tertiary-medium)"/>
+<rect x="113" y="106.626" width="29.5091" height="65.3415" rx="6" transform="rotate(-47.1283 113 106.626)" fill="var(--tertiary-high)"/>
+<circle cx="73" cy="47" r="37.5" fill="var(--tertiary-medium)" stroke="var(--tertiary-high)"/>
+<circle cx="60.5" cy="36.5" r="4" fill="var(--secondary)" stroke="var(--secondary)"/>
+<circle cx="85.5" cy="36.5" r="4" fill="var(--secondary)" stroke="var(--secondary)"/>
+<path d="M58 61.8011C66.2918 53.492 75.574 55.8621 83 63.0002" stroke="var(--secondary)" stroke-width="2"/>
+<path d="M200.473 105.782C200.473 95.2311 201.557 82.8566 193.945 74.5196C189.225 69.3502 174.712 64.2654 174.712 64.2654C163.984 62.2082 157.391 67.0725 163.108 77.871C165.857 83.0642 172.298 87.4386 177.964 88.4503C186.94 90.0531 198.843 86.3663 205.425 80.1219C214.619 71.3993 223.721 36.595 211.728 27" stroke="var(--tertiary-high)" stroke-width="2" stroke-linecap="round" stroke-dasharray="3 4"/>
+</svg>
+</div>
+
+<div class="title_wrapper">
   <h1 class="title"><%= @title %></h1>
+  <p class="subtitle"><%= @subtitle %></p>
+  <a class="btn btn-primary btn-large --home" href="/"> <%= SvgSprite.raw_svg('house') %> <%= I18n.t('page_not_found.home') %>
```

Comment:
We should use `Discourse.base_url` for the `href` here, this will handle subfolder sites. Ideally we would use the `defaultHomepage()` functionality in JS to get the site/user's homepage (since it can be set to `top` and so on), but that's not accessible here